app/views/content/shared/teaching/_support.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="support">
```

Comment:
Minor suggestion - I would be tempted to rename this partial/class, perhaps to `ect_support` @Sarah-DfE @joegibb @sarahcrack 

We normally use the term support to refer to our GIT support services (emails, events, advisers, contact centre, etc.) whereas this is very explicitly talking about support when you are a teacher

I'm sure we will have more partials in the future that refer to GIT support services so might be worth renaming this one now to avoid confusion in future?

NB the content within the partial is absolutely fine and clear - I'm just thinking in terms of maintenance in future!